practice-three/src/js/script.js

```diff
@@ -240,7 +242,7 @@ const renderUserTable = () => {
           fetch(userApi + '/' + registrationForm.getAttribute('data-id'), options)
             // Parses JSON response into native JavaScript objects 
             .then((response) => response.json())
-            .then()
+            .then(renderUserTable())
```

Comment:
Should be .then( () => renderUserTable()) or .then(() => { renderUserTable(); }). Pls check another code